CHANGES

```diff
@@ -1,6 +1,7 @@
 0.191
 
 - Do not require keystore password for HTTPS.
+- Require host to be set in HttpUriBuilder.
```

Comment:
do you want to indicate this is backwards-incompatible change?

Reply:
We don't typically go into that much detail for small changes. I think it's clear from the description that it will now fail in this (edge) case.